laCour/slack-night-mode
# add highlight-color to new mention classes

Slack began using the ".c-mrkdwn__*mention" class for highlighting
mention words (including @channel and @here), this caused the default
theme color to become unreadable in those blocks.

This commit simply sets the color for these new mention classes, but
only underneath "c-message_body".

Before (broken): https://cl.ly/3O3y2I1z1j3f
After (fixed): https://cl.ly/3E36171B1Z0s

References #139

Test Plan:
- Have one or more highlight words configured in Slack
- Use one of those words in a message
- Verify that the text color is readable on slacks yellow-ish background
- Do the same test for an @channel, @here, and @user
